apis/io-engine/protobuf/v1/common.proto

```diff
@@ -7,3 +7,32 @@ enum ShareProtocol {
   NVMF = 1; // NVMe over Fabrics (TCP)
   ISCSI = 2; // iSCSI
 }
+
+// Common IO stats struct for all resource types.
+message IoStats {
+  oneof iostat {
+    StatInTicks stat_ticks = 1; // Struct with Latency represented in ticks.
+    StatInInterval stat_interval = 2; // Struct with Latency represented in interval.
```

Comment:
"in interval" sounds a bit funny
Should it be more like stat_time_interval?
```suggestion
    StatInInterval stat_time_interval = 2; // Latency represented in time intervals.
```

Reply:
changed it to stat_as_interval and stat_as_ticks. Is that okay?